cfme/containers/project.py

```diff
@@ -89,7 +89,7 @@ def step(self):
                                                    use_search=True).click()
 
     def resetter(self):
-        if self.appliance.version == '5.9':
+        if self.appliance.version.is_in_series([5, 9]):
```

Comment:
**REQUIRED**: The usual use for this is ```.is_in_series('5.9')```.  Why do we have an array here?

Reply:
well, it works, but I'll change :+1: